internal/provisioner/plain/controllers/bundleinstance_controller.go

```diff
@@ -143,6 +145,11 @@ func (r *BundleInstanceReconciler) Reconcile(ctx context.Context, req ctrl.Reque
 		Metadata: &chart.Metadata{},
 	}
 	for _, obj := range desiredObjects {
+		// Add required crdvalidator annotation if it is a CRD
+		if obj.GetObjectKind().GroupVersionKind().GroupVersion() == apiextensionsv1.SchemeGroupVersion {
+			crdanno.Add(obj)
+		}
```

Comment:
I think I'd be slightly surprised as a user with this UX where we inject runtime annotations to objects but I won't block a PR over this kind of behavior.

Reply:
I agree with you. Let me try an implementation where we look at the owner reference for incoming CRDs and only run the `crdvalidator` against that.

Reply:
Okay, the most recent PR added this functionality. Now the provisioner is not adding anything and instead the `crdvalidator` is watching the `OwnerReference` of incoming CRDs. If it is owned by Rukpak or has the annotation set  to `enabled` it will run the validation. If the CRD has the annotation set as `disabled`, it will not run the validation. This keeps it so we're not editing annotations and have a valve for opting out.